pkgs/by-name/ko/komac/package.nix

```diff
@@ -2,6 +2,7 @@
   lib,
   stdenv,
   fetchFromGitHub,
+  fetchpatch,
```

Comment:
Ah, this line looks like a removed code?

Reply:
Forgot to remove it 😅